src/share/classes/com/sun/javatest/regtest/tool/i18n.properties

```diff
@@ -260,6 +260,8 @@ help.main.nativepath.arg=<path>
 help.main.nativepath.desc=Path to location of native libraries and programs needed \
     by the tests.
 
+help.main.crashonly.desc=Ignores regular failures and only reports crashes as failures.
```

Comment:
What  it does with passes? they are reported as pass even if there si crash, right? So maybe
```
Only consider test as failed if it was both failed and JVM crashed during execution
```

wdyt?

Reply:
hm, right.. that sounds better honestly :D